src/evp/evp.py

```diff
@@ -43,5 +44,9 @@ def estimate_value(self, lead_id) -> LeadValue:
         )
         get_database().update_lead(lead)
 
+        # saving the trained models
+        joblib.dump(self.probability_predictor, 'probability_model.joblib')
+        joblib.dump(self.life_time_value_predictor, 'life_time_value_model.joblib')
```

Comment:
Maybe we can add a rule to the .gitignore for *.joblib files, as we need a license for them otherwise as well. We probably also don't want the models in the repo right?

Reply:
Very good point, will adjust gitignore file accordingly